Sources/GRPC/ClientConnection.swift

```diff
@@ -101,6 +101,9 @@ public class ClientConnection {
   }
 
   /// Creates a new connection from the given configuration.
+  ///
+  /// - Important: Users should prefer using `ClientConnection.secure(group:)` to build a connection
+  ///   with TLS, or `ClientConnection.insecure(group:)` to build a connection without TLS.
   public init(configuration: Configuration) {
```

Comment:
This is still alpha right? You *could* make an API-breaking change to hide these initialisers and *force* people to use the new `secure` and `insecure`?

Reply:
That's right but it's impossible to provide fix-its for this and it's quite an annoying change to have to make. I'm leaning towards keeping it for now. The longer term plan would be to not add anything to it to gently force people to the builder.

Reply:
Please do not hide this initialiser, as this would disallow precise configuration by hands, which is often needed.

Reply:
@be4rd what's your use case here? Everything should be representable by the builder as well.

Reply:
@glbrntt if you'd like to keep it another option to dissuade use would be to deprecate or provide a warning for direct use

Reply:
@glbrntt sorry, did not saw TLS configuration methods in builder when I left a comment 😅